File: SFML-PONG/src/paddle.cpp
#include "../include/Paddle.h"

Paddle::Paddle(int WIDTH, int HEIGHT, sf::Color COLOR):
    Rectangle(WIDTH,HEIGHT,COLOR)
{
    //ctor
}

Paddle::~Paddle()
{
    //dtor
}

void Paddle::goUp()
{

    Object.move(0, -0.20);
}

void Paddle::goDown()
{

    Object.move(0, 0.20);
}

File: JSFML-PONG/src/pong/Paddle.java
package pong;

import org.jsfml.graphics.Color;

public class Paddle extends Rectangle {

	public Paddle(int WIDTH, int HEIGHT, Color COLOR) {
		super(WIDTH, HEIGHT, COLOR);
	}

	void goUp() {
		Object.move(0, -0.2f);
	}

	void goDown() {
		Object.move(0, 0.2f);
	}

}
